manifests/plugin.pp

```diff
@@ -77,7 +77,7 @@
     $proxy_port  = undef,
 ) {
 
-  include elasticsearch
+  require elasticsearch
```

Comment:
Thanks Mr. Jenkins for spotting that ;)

I tested it for me, and did not ran into recursion issues, therefore kept it.
but I also don't have to deal with that augeas thingie bingie for that defaults file on OpenBSD.

reverted it back to 'include', since the include also seems to be fine for me.